src/cli.js

```diff
@@ -632,7 +638,12 @@ var exports = {
         mergeCLIPrereq(config);
 
         lint(extract(code, opts.extract), results, config, data, filename);
-        (opts.reporter || defReporter)(results, data, { verbose: opts.verbose });
+
+        // invoke each reporter
+        for (var index in opts.reporters) {
+          (exports.getReporter(opts.reporters[index]))(results, data, { verbose: opts.verbose });
```

Comment:
Can you remove the parentheses around `exports.getReporter(...)`? (Just for style consistency)